lara-typescript/src/section-authoring/components/section-item-picker.tsx

```diff
@@ -134,12 +148,11 @@ export const SectionItemPicker: React.FC<IProps> = (props) => {
         {allItemsList.map((item, index) => {
           const isSelectedItem = currentSelectedItem === item;
           const itemClass = setItemClasses(isSelectedItem);
-          const itemDisabled = itemSelected && !isSelectedItem ? true : false;
           return (
             <li key={`ai-${index}`}>
               <SectionItemButton
                 item={item}
-                disabled={itemDisabled}
+                disabled={false}
```

Comment:
Seems like the `disabled` property of SectionItemButton is also always going to be false. So maybe we can do away with that property?

Reply:
For now, I am just going to leave the components defined as they are because I cant rule out a use case later.